front/app/modules/commercial/flag_inappropriate_content/citizen/components/NLPFlagNotification.tsx

```diff
@@ -19,7 +19,7 @@ const NLPFlagNotification = memo<Props>((props) => {
 
   return (
     <NotificationWrapper
-      linkTo={notification.attributes.flaggable_url}
+      linkTo={`/${notification.attributes.flaggable_path}`}
```

Comment:
I chose to add the leading `/` here, rather than in the BE `def model_to_path`, since conventionally relative paths do not start with a leading `/`. However, I have no strong feelings for or against either approach.

Reply:
Thanks for pointing that out, I might have missed it.

I think you should make it start with a slash, because relative paths (the way we are using them here) _not_ really relative patha, but apparently called `absolute-path references`. If found that nice term in [this SO thread](https://webmasters.stackexchange.com/questions/56840/what-is-the-purpose-of-leading-slash-in-html-urls).